### PR TITLE
JENKINS-37202# Set pagination by default

### PR DESCRIPTION
**Pull request by vivek**
# Description

See https://issues.jenkins-ci.org/browse/JENKINS-37202.

* Create 120 jobs (?)
* Call URL localhost:8080/jenkins/blue/rest/search/?q=type:pipeline
* You should get only 100 records

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [x] Ran Acceptance Test Harness against PR changes.
Ran ATH locally, all tests passed successfully.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@jenkinsci/code-reviewers @reviewbybees

### CODE REVIEW

**Comment by imeredith** [comment]
:bee: lgtm

File: blueocean-rest/src/main/java/io/jenkins/blueocean/rest/pageable/PagedResponse.java [hunk 1]
```diff
@@ -28,6 +28,7 @@
 @InterceptorAnnotation(PagedResponse.Processor.class)
 public @interface PagedResponse {
     class Processor extends Interceptor {
+        private static final int DEFAULT_LIMIT=100;
```
Comment (tfennelly) [inline blueocean-rest/src/main/java/io/jenkins/blueocean/rest/pageable/PagedResponse.java]: Hey Vivek, is this default configurable in any way? Would it make sense to allow people set a startup prop (with an upper and lower bound)?

**Comment by tfennelly** [comment]
🐝 LGTM

File: blueocean-rest/src/main/java/io/jenkins/blueocean/rest/pageable/PagedResponse.java (hunk 1 above)
Comment (vivek) [inline blueocean-rest/src/main/java/io/jenkins/blueocean/rest/pageable/PagedResponse.java]: @tfennelly Not at the moment. We can set it via property if need be but please note that limit query parameter lets you override it as well. What makes it complicated is that with https://issues.jenkins-ci.org/browse/JENKINS-38055, we are going to revisit it to keep per API limit default.

**Pull request merged**